usr/sbin/pkcsep11_migrate/pkcsep11_migrate.c

```diff
@@ -100,13 +97,11 @@ static int reencrypt(CK_SESSION_HANDLE session, CK_ULONG obj, CK_BYTE *old)
     target_t target = XCP_TGT_INIT;
     CK_RV rc;
     CK_BYTE name[256];
-
-    ep11_opaque *op_old = (ep11_opaque *) old;
-    ep11_opaque op_new;
-
+    unsigned char blob[blobsize];
+    CK_ULONG blob_len;
 
     CK_ATTRIBUTE opaque_template[] = {
-        {CKA_IBM_OPAQUE, &op_new, sizeof(op_new)}
+        {CKA_IBM_OPAQUE, blob, sizeof(blob)}
```

Comment:
I struggled over: blob is a pointer but sizeof(blub) is the size of blob[]. Can we make this clear with using BLOBSIZE instead ?

Reply:
fixed